x-pack/plugins/observability/public/index.ts

```diff
@@ -82,8 +83,11 @@ export { observabilityFeatureId, observabilityAppId } from '../common';
 export { useChartTheme } from './hooks/use_chart_theme';
 export { useBreadcrumbs } from './hooks/use_breadcrumbs';
 export { useTheme } from './hooks/use_theme';
+export { useKibanaSpace } from './hooks/use_kibana_space';
```

Comment:
I would move more hooks to Obs-Shared, not more to Obs.

Reply:
thanks for catching it, i have removed it.